torch/testing/_internal/common_utils.py

```diff
@@ -940,7 +881,9 @@ def read_and_process():
                     f"Test is disabled because an issue exists disabling it: {issue_url}" +
                     f" for {'all' if platforms == [] else ''}platform(s) {', '.join(platforms)}." +
                     " To enable, set the environment variable PYTORCH_RUN_DISABLED_TESTS=1")
-
+    if TEST_SKIP_FAST:
```

Comment:
I think the logic here is a bit changed because of the changes around lines 866. Could you group this logic with that one as they're both about slowness?